### ISSUE TITLE
Update of the NPM package needed (Belgian BIC's are updated)

### ISSUE THREAD

**Issue by sam-medispring** [sigalor/iban-to-bic#5]
Hello,
Is it possible to publish a new version of the npm package with an update dataset ?

**Comment by sigalor** [comment]
Of course, feel free to create a PR with the updated datasets! Don't really have time to work on this myself. Possibly the parsers/scrapers also need to be updated.

**Comment by sam-medispring** [comment]
Thx !
https://github.com/sigalor/iban-to-bic/pull/6

I updated the DE scraper, i did a check on the resulting JSON, they seem good :-)
